espnet2/text/korean_cleaner.py

```diff
@@ -0,0 +1,77 @@
+
+## Referenced from https://github.com/hccho2/Tacotron-Wavenet-Vocoder-Korean
+
+import re
+
+class Korean_cleaner:
+    @classmethod
+    def _normalize_numbers(cls, text):
+        number_to_kor = {
+            '0': '영',
+            '1': '일',
+            '2': '이',
+            '3': '삼',
+            '4': '사',
+            '5': '오',
+            '6': '육',
+            '7': '칠',
+            '8': '팔',
+            '9': '구',
+        }
+        new_text = "".join(number_to_kor[char] if char in number_to_kor.keys() else char for char in text)
+        return new_text
+
+    @classmethod
+    def _normalize_english_text(cls, text):
+        upper_alphabet_to_kor = {
+            'A': '에이',
+            'B': '비',
+            'C': '씨',
+            'D': '디',
+            'E': '이',
+            'F': '에프',
+            'G': '지',
+            'H': '에이치',
+            'I': '아이',
+            'J': '제이',
+            'K': '케이',
+            'L': '엘',
+            'M': '엠',
+            'N': '엔',
+            'O': '오',
+            'P': '피',
+            'Q': '큐',
+            'R': '알',
+            'S': '에스',
+            'T': '티',
+            'U': '유',
+            'V': '브이',
+            'W': '더블유',
+            'X': '엑스',
+            'Y': '와이',
+            'Z': '지',
+        }
+        new_text = re.sub('[a-z]+', text.upper(), text)
+        new_text = "".join(upper_alphabet_to_kor[char] if char in upper_alphabet_to_kor.keys() else char for char in new_text)
+
+        return new_text
+
+
+    @classmethod
+    def normalize_text(cls, text):
+        # stage 0 : text strip
+        text = text.strip()
+
+        # stage 1 : normalize numbers
+        text = cls._normalize_numbers(text)
+
+        # stage 2 : normalize english text
+        text = cls._normalize_english_text(text)
+        return text
+
+if __name__ == '__main__':
+    sentence = "내가 ABC 마트에 가는길은 너무나 험난해. 1분 1초도 아까워"
+    sentence = "지난 mbC 프로그램 봤어? 12세 이상 관람가인데 이상해"
+
+    new_sentence = Korean_cleaner.normalize_text(sentence)
+    print(new_sentence)
```

Comment:
Could you move this test to the following test code?
https://github.com/espnet/espnet/blob/master/test/espnet2/text/test_cleaner.py

Reply:
Sure I can. 

Btw I have a question regarding space_symbol. "PhonemeTokenizer" class takes "space_symbol" argument but not sure where it changed " " to space_symbol defined. tested on korean gp2k for token list generation and " " was put in token list instead of <space>.  I converted " " to "<space>" in "Jaso" class I created as an workaround. Because I don't know how other g2p modules work, if you think its okay, I can add a line that converts " " to space_symbol in PhonemeTokenizer.text2tokens.

Reply:
Oh you are right. Space symbol is not used in the current setting, so " " is used as is. Let me think it.

Reply:
Okay. I will hold it till its determined then.

Reply:
Sorry for the late reply.
I think replacing with `<space>` in your class should be fine (also using " " as is also fine). So keeping current style is OK.
The modification of `PhonemeTokenizer.text2tokens` might affect the compatibility of the pretrained model, so please keep it as is for now.

Reply:
Sorry for the late reply.
I think replacing with `<space>` in your class should be fine (also using " " as is also fine). So keeping current style is OK.
The modification of `PhonemeTokenizer.text2tokens` might affect the compatibility of the pretrained model, so please keep it as is for now.

Reply:
okay! thank you. then ill proceed the remaining process.